Xamarin.Forms.Platform.Android/AppCompat/NavigationPageRenderer.cs

```diff
@@ -541,6 +541,8 @@ void RemovePage(Page page)
 
 		void ResetToolbar()
 		{
+			AToolbar oldToolbar = _toolbar;
```

Comment:
Since this is a local variable, it should go out of scope and not cause a memory leak. `_toolbar` will represent a different object in memory after it's set to null (a few lines below) and new-ed up.